src/main/java/sirius/biz/jdbc/DatabaseController.java

```diff
@@ -95,6 +98,12 @@ public void executeQuery(WebContext ctx, JSONStructuredOutput out) throws SQLExc
             String sqlStatement = ctx.get("query").asString();
             SQLQuery qry = db.createQuery(sqlStatement);
 
+            if (!Sirius.isDev()) {
```

Comment:
gabs dazu nicht rückfragen in nem anderen pr?

Reply:
öhm, ich wüsste nicht wo? ist kein anderer PR offen

Reply:
jup